src/java.base/share/classes/jdk/internal/reflect/NativeConstructorAccessorImpl.java

```diff
@@ -28,14 +28,20 @@
 import sun.reflect.misc.ReflectUtil;
 
 import java.lang.reflect.*;
+import jdk.internal.misc.Unsafe;
 
 /** Used only for the first few invocations of a Constructor;
     afterward, switches to bytecode-based implementation */
 
 class NativeConstructorAccessorImpl extends ConstructorAccessorImpl {
+    private static final Unsafe unsafe = Unsafe.getUnsafe();
```

Comment:
`static final` field identifiers should be capitalized. Suggestion: `private static final Unsafe U = Unsafe.getUnsafe()`.

Reply:
fixed